src/System.Slices/src/System/Span.cs

```diff
@@ -335,7 +320,7 @@ public override bool Equals(object obj)
         /// gain: performance: no boundaries check (single operation) 
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private T GetItemWithoutBoundariesCheck(int index)
+        internal T GetItemWithoutBoundariesCheck(int index)
```

Comment:
If we need it internally to implement extension method or anything then sounds like it should it be public as the public surface is insufficient to implement stuff (since this is incredibly unsafe method I would add unsafe in the name). cc: @KrzysztofCwalina